components/org.wso2.carbon.secvault.securevault/src/main/java/org/wso2/carbon/secvault/securevault/MasterKey.java

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-package org.wso2.carbon.securevault;
+package org.wso2.carbon.secvault.securevault;
 
 import java.util.Optional;
 
 /**
  * This class represents a master key that is needed to initialize a secret repository. A secret repository might need
  *  a few master keys to get it initialized.
  *
- * @since 5.2.0
+ * @since 1.0.0
```

Comment:
Are we planning to use v1.0.0 for the next release? I think it would be better not to reset the version value as secure vault was release earlier together with the kernel.